app/admin/views.py

```diff
@@ -231,3 +231,24 @@ def change_agency_public_status(agency_id):
     form.process()
     return render_template('admin/manage_agency.html', agency=agency,
                            form=form)
+
+
+@admin.route('/_update_editor_contents', methods=['POST'])
```

Comment:
I learned the style from @weronica but I think it's a good practice so it's obvious which routes are not meant to be accessed by the user